src/amber/cli/commands/database.cr

```diff
@@ -41,6 +41,11 @@ module Amber::CLI
 
       def run
         CLI.toggle_colors(options.no_color?)
+
+        if args.commands.empty?
```

Comment:
I don't know how I feel about defaulting to connect to the db when no arguments are present.

Reply:
I'm assuming that this default would be invoked with simply `amber db`? If so, I see no problem with that.